docs/segment.md

```diff
@@ -2,98 +2,188 @@
-- You can also have the option to calculate the masks en masse in the Images tab -> segment -> choose 'Only masks'.
+### Segmenting text lines with eScriptorium
 
-- ++left-button++ on a line to select it, then you can drag its closest control point.
-- **Double click** on the line will create a new control point at the mouse location.
-- You can reverse the order of the line's points by selecting a line (or multiple) and using reverse (++i++).
-- When in region mode, Regions can be edited the same way.
-Selected lines can be linked or unlinked to regions with the corresponding buttons (++y++) (++u++).
-- By using the cut mode (++c++) you can cut through lines, splitting them in two or removing part of them.
-- If at least two lines are selected a option to join them becomes available (++j++).
-You can go through your changes history back and forth with ++ctrl+z++ (undo) and ++ctrl+Y++ (redo) or by using the corresponding buttons .
+When segmenting text lines with eScriptorium, you will generally want to draw baseline. If you are in text region mode, the **mosaic** button should be in green meaning it is activate. Unclick it, or just press ++r++. The text regions should now just be the outlines of the different bounding boxes.
 
-- ++shift+left-button++ allows to add or remove a line from the selection,
-- Clicking on an empty space clears the selection,
-- ++shift++ and drag creates a lasso selection tool that allows to select control points (they then appear black).
+To segment text lines, you can:
 
-- **Note:** If lines are already selected the lasso will only select points in those lines.
+- Click and drag to underline a line of text. It is similar to a freehand drawing mode. It is generally not recommended, unless you have a specific use case.
 
-- ++ctrl++ and drag allows to move all the selected control points at once (or the selected lines if no control points are selected).
+![image: Drawing a baseline freehand.](img/segment/baseline_freehand.gif "You can freehand draw baselines on eScriptorium, however it is not recommended.")
 
-- The red trash button deletes all selected lines/regions,
-- The yellow trash button only deletes selected control points.
+- We recommend to segment text lines by clicking on the beginning of your text lines, and then clicking where you want your text line to end. The text line will be straight.
 
-<!--## Editing lines
+![image: Drawing a straight baseline.](img/segment/baseline_straight.gif "You can draw straight baselines.")
 
-## Editing regions -->
+The mask of the segmented text line is then automatically calculated by eScriptorium by clicking on the **thumbs up** button. You can display three views of the baselines with the **toggle line masks and stroke width** button, or by pressing ++m++. One of them will let the mask appear.
 
-## Reordering
+eScriptorium also calculate the mask associated with each line asynchronously, you do not need to always press the dedicated button.
```

Comment:
"calculates"
I think you could make it a tip rather than a simple paragraph

Reply:
not necessarily a tip, but this line could be regrouped with the previous one. They say the same thing.